doc/cephadm/certmgr.rst

```diff
@@ -103,54 +111,54 @@ Cephadm certmgr supports three different scopes for certificate management:
 
 1. **Global Scope:**
```

Comment:
Suggest to change these 3 items into an ordered list instead of hardcoding the numbers.

```suggestion
#. **Global Scope:**
```

And similarly to lines 116, 121.

Reply:
Fixed